src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Common/VSInternalServerCapabilitiesExtensions.cs

```diff
@@ -43,4 +43,9 @@ public static void EnableValidateBreakpointRange(this VSInternalServerCapabiliti
     {
         serverCapabilities.BreakableRangeProvider = true;
     }
+
+    public static void EnableMapCodeProvider(this VSInternalServerCapabilities serverCapabilities)
```

Comment:
I just did this for when we have cohost mapcode, but I was too lazy to migrate it all at the moment

Reply:
No, you did this because this is now _the_ way we do capabilities :P

There isn't a lot of value to porting more things to cohosting at the moment anyway.